src/main/java/service/impl/ReportGeneratorImpl.java

```diff
@@ -0,0 +1,21 @@
+package service.impl;
+
+import db.Storage;
+import java.util.Map;
+import service.ReportGenerator;
+
+public class ReportGeneratorImpl implements ReportGenerator {
+    @Override
+    public String getReport() {
+        StringBuilder stringBuilder = new StringBuilder();
+        stringBuilder.append("fruit,quantity").append(System.lineSeparator());
+
+        for (Map.Entry<String, Integer> pair : Storage.reports.entrySet()) {
```

Comment:
Direct access to the Storage class' reports field violates the encapsulation principle. Consider adding a method in the Storage class to retrieve the data needed for the report.